pb/protos/profile.proto

```diff
@@ -14,34 +14,27 @@ message Profile {
     string phoneNumber                     = 8;
     repeated SocialAccount social          = 9;
```

Comment:
`website`, `email`, `phoneNumber` and `social` all semantically fall under `Contact`. Could bundle similarly to `Colors` and `Stats`.